R/pr.R

```diff
@@ -93,11 +93,7 @@ pr_fetch <- function(number,
   check_uses_github()
   check_uncommitted_changes()
 
-  auth_token <- github_token()
-  check_github_token(auth_token)
-
-  protocol <- github_remote_protocol()
-  credentials <- git2r_credentials(protocol, auth_token)
+  auth_token <- maybe_github_token()
```

Comment:
I think this should be moved to the arguments for consistency with the other functions

Reply:
Two weeks ago you said:
> I would rather not add these arguments, if the only argument is for consistency. I think we should be moving away from these arguments in general, in favour of better tooling for declaring credentials globally.

and I agree with Past Hadley 😉

https://github.com/r-lib/usethis/pull/656#discussion_r266192941